sdk/python/responsible-ai/text/README.md

```diff
@@ -0,0 +1,48 @@
+
+## Directory 📖
+ 
+
+
+
+| Scenario | Dataset | Data type | RAI component included | Link to sample | Documentation |
+| --- | --- | --- | --- | --- | --- |
+| Text Classification | [DBPedia dataset](https://huggingface.co/datasets/DeveloperOats/DBPedia_Classes) | Text | Explanation, Error Analysis | [responsibleaidashboard-text-classification-DBPedia.ipynb](./text/responsibleaidashboard-text-classification-DBPedia.ipynb) | [Text Dashboard Generation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-responsible-ai-text-dashboard?view=azureml-api-2) |
+| Multi-label Text Classification | [Covid19 Emergency Event Dataset](https://huggingface.co/datasets/joelito/covid19_emergency_event) | Text | Explanation, Error Analysis | [responsibleaidashboard-multilabel-text-classification-covid-events.ipynb](./text/responsibleaidashboard-multilabel-text-classification-covid-events.ipynb) | [Text Dashboard Generation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-responsible-ai-text-dashboard?view=azureml-api-2) |
+| Binary Text Classification | [blbooksgenre dataset](https://huggingface.co/datasets/blbooksgenre) | Text | Explanation, Error Analysis | [responsibleaidashboard-text-classification-blbooksgenre.ipynb](./responsibleaidashboard-text-classification-blbooksgenre.ipynb) | [Text Dashboard Generation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-responsible-ai-text-dashboard?view=azureml-api-2) |
+| Text Classification | [News Article Classification](./text/responsibleaidashboard-text-classification-financial-news/Text_classification_dataset.xlsx) | Text | Explanation, Error Analysis | [Financial_News_Text_classifier.ipynb](./text/responsibleaidashboard-text-classification-financial-news/responsibleaidashboard-text-classification-financial-news.ipynb) | [Documentation](./text/responsibleaidashboard-text-classification-financial-news/readme.md) |
+| Text Question Answering | [Stanford Question Answering Dataset (SQuAD)](https://huggingface.co/datasets/squad) | Text | Explanation, Error Analysis | [responsibleaidashboard-text-question-answering-squad.ipynb](./text/responsibleaidashboard-text-question-answering-squad.ipynb) | [Text Dashboard Generation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-responsible-ai-text-dashboard?view=azureml-api-2)|
+
+
+To learn more about the different types of Dashboard visit the tutorial -
+[Text Dashboard Generation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-responsible-ai-text-dashboard?view=azureml-api-2)
+
+
+## Supportability 🧰
+Currently, we support datasets having numerical and categorical features. The following table provides the scenarios supported for each of the four responsible AI components:
```

Comment:
Is this true for text dashboards? I guess not. Maybe remove it.

Reply:
yes any additional metadata features with categorical and numeric values are supported - but the dashboard itself runs on freeform text